maven-enforcer-plugin/src/main/java/org/apache/maven/plugins/enforcer/EnforceMojo.java

```diff
@@ -97,6 +97,12 @@
     @Parameter( property = "enforcer.failFast", defaultValue = "false" )
     private boolean failFast = false;
 
+    /**
+     * Set this to "true" to cause a failure if there are no rules to execute
+     */
+    @Parameter( property = "enforcer.failIfNoRules", defaultValue = "true" )
```

Comment:
Pleas add `@since`

Yes - I see that other parameters also has missing it ... we can fix for rest in separate PR

It is used for  documentation: https://maven.apache.org/enforcer/maven-enforcer-plugin/enforce-mojo.html

Maybe documentation - similar to other, eg:

```
Flag to allow executions without rules.
```